plasma_framework/contracts/src/framework/ExitGameController.sol

```diff
@@ -39,89 +41,97 @@ contract ExitGameController is ExitGameRegistry {
+        bytes32 key = exitQueueKey(vaultId, token);
+        require(hasExitQueue(key), "Such token has not been added to the plasma framework yet");
+        PriorityQueue queue = exitsQueues[key];
 
-        uint256 uniquePriority = ExitPriority.computePriority(_exitableAt, _txPos, _exitId);
+        uint256 uniquePriority = ExitPriority.computePriority(exitableAt, txPos, exitId);
 
         queue.insert(uniquePriority);
-        delegations[uniquePriority] = _exitProcessor;
+        delegations[uniquePriority] = exitProcessor;
 
-        emit ExitQueued(_exitId, uniquePriority);
+        emit ExitQueued(exitId, uniquePriority);
         return uniquePriority;
     }
 
     /**
      * @notice Processes any exits that have completed the challenge period. Exits would be processed according to the exit priority.
      * @dev emits ProcessedExitsNum event.
-     * @param _token token type to process.
-     * @param _topExitId unique priority of the first exit that should be processed. Set to zero to skip the check.
-     * @param _maxExitsToProcess maximal number of exits to process.
+     * @param vaultId vault id of the vault that stores exiting funds.
+     * @param token token type to process.
+     * @param topExitId unique priority of the first exit that should be processed. Set to zero to skip the check.
+     * @param maxExitsToProcess maximal number of exits to process.
      * @return total number of processed exits
      */
-    function processExits(address _token, uint160 _topExitId, uint256 _maxExitsToProcess) external {
-        require(hasToken(_token), "Such token has not been added to the plasma framework yet");
-
-        PriorityQueue queue = exitsQueues[_token];
+    function processExits(uint256 vaultId, address token, uint160 topExitId, uint256 maxExitsToProcess) external {
+        bytes32 key = exitQueueKey(vaultId, token);
+        require(hasExitQueue(key), "Such token has not been added to the plasma framework yet");
+        PriorityQueue queue = exitsQueues[key];
         require(queue.currentSize() > 0, "Exit queue is empty");
 
         uint256 uniquePriority = queue.getMin();
         uint160 exitId = ExitPriority.parseExitId(uniquePriority);
-        require(_topExitId == 0 || exitId == _topExitId,
+        require(topExitId == 0 || exitId == topExitId,
             "Top exit id of the queue is not the same as the specified one");
 
         IExitProcessor processor = delegations[uniquePriority];
         uint256 processedNum = 0;
 
-        while (processedNum < _maxExitsToProcess && ExitPriority.parseExitableAt(uniquePriority) < block.timestamp) {
+        while (processedNum < maxExitsToProcess && ExitPriority.parseExitableAt(uniquePriority) < block.timestamp) {
             delete delegations[uniquePriority];
             queue.delMin();
             processedNum++;
 
-            processor.processExit(exitId, _token);
+            processor.processExit(exitId, token);
```

Comment:
I think we should change the interface for `processExit` too. So we should call with `processExit(exitId, vaultId, token)`.

Reply:
done